### PR TITLE
Improve remediation of auditd_data_disk_full_action

### PR DESCRIPTION
**Pull request by matejak**
So far, I have made the remediation to use our shared remediation functionality. However, the remediation is not included in the datastream. I have no idea why, renaming the bash file to `content_rule_auditd_data_disk_full_action.sh` didn't help. 
Any ideas?

### CODE REVIEW

**Comment by redhatrises** [comment]
```
# Include source function library.
. /usr/share/scap-security-guide/remediation_functions
```

**Comment by matejak** [comment]
I have amended my commit, so the script contains the exact comment. The inclusion of remediation functions was there even before.
Next, I have decided to remove other comments that just stated the obvious.

File: shared/fixes/bash/auditd_data_disk_full_action.sh [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
 . /usr/share/scap-security-guide/remediation_functions
+
 populate var_auditd_disk_full_action
 
-#
-# If disk_full_action present in /etc/audit/auditd.conf, change value
-# to var_auditd_disk_full_action, else
-# add "disk_full_action = $var_auditd_disk_full_action" to /etc/audit/auditd.conf
-#
+cpe_number='27370-6'
```
Comment (redhatrises) [inline shared/fixes/bash/auditd_data_disk_full_action.sh]: why this change?

Comment (matejak) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: I think that it increases readability at no cost.

Comment (redhatrises) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: In this shorthand script, sure. But in HTML output, I don't think that it does.

Comment (mpreisler) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: we have a macro for CCEs, they must not be hardcoded in the scripts

Comment (mpreisler) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: and it's cce, not cpe, so this makes it super confusing

Comment (redhatrises) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: If I recall correctly, making this change would break that macro, but I haven't tried it to confirm.

Comment (matejak) [reply shared/fixes/bash/auditd_data_disk_full_action.sh]: OK, I have rebased and put the `@CCENUM@` macro back there.

**Comment by matejak** [comment]
To explain what went wrong, I have accidentally mixed up this remediation with remediation for another rule, therefore I couldn't find it when I was looking for it (this is a RHEL6 remediation, whereas I was looking for it in RHEL7).
As of now, nothing seems to be wrong with it.

**Comment by yuumasato** [comment]
Ack.
